rotkehlchen/tests/unit/test_eth2.py

```diff
@@ -412,28 +427,31 @@ def test_validator_daily_stats_with_db_interaction(  # pylint: disable=unused-ar
 def test_validator_daily_stats_with_genesis_event(
         network_mocking: bool,
         price_historian: 'PriceHistorian',  # pylint: disable=unused-argument
+        database: DBHandler,
+        messages_aggregator: MessagesAggregator,
 ) -> None:
     """
     Test that if profit returned by beaconchain on genesis is > 32 ETH the app
     handles it correctly.
     """
     validator_index = 999
 
-    with mock_scrape_validator_daily_stats(network_mocking):
-        stats = scrape_validator_daily_stats(
+    beaconchain = BeaconChain(database, messages_aggregator)
+    with mock_query_validator_daily_stats(beaconchain, network_mocking):
+        stats = beaconchain.query_validator_daily_stats(
             validator_index=validator_index,
             last_known_timestamp=Timestamp(0),
             exit_ts=None,
         )
     assert stats == [
         ValidatorDailyStats(
             validator_index=999,
-            timestamp=DAY_AFTER_ETH2_GENESIS,
-            pnl=FVal('0.01201'),
+            timestamp=Timestamp(DAY_AFTER_ETH2_GENESIS + 23),  # beaconchain shifts it 23 seconds
```

Comment:
Hmm this is not okay. You probably need to adjust the definition of this. As the only place it was used it's compared with `==`.

```python
            elif column_pos == 2:
                pnl = _parse_fval(column.string, 'income')
                # if the validator makes profit in the genesis day beaconchain returns a
                # profit of deposit + validation reward. We need to subtract the deposit value
                # to obtain the actual pnl.
                # Example: https://beaconcha.in/validator/999/stats
                if pnl > ONE and timestamp == DAY_AFTER_ETH2_GENESIS:
                    pnl -= INITIAL_ETH_DEPOSIT

                column_pos += 1
```

It only used there, in the tests now and in DB upgrades. And in the DB upgrade it should have the old value as it depended on that. So really as I look at it this constant can go away? And be hard coded in the db upgrade to the old value?